test/quantization/core/test_quantized_op.py

```diff
@@ -2585,6 +2588,10 @@ def forward(
 
                     # Prepare
                     mha.qconfig = torch.ao.quantization.get_default_qconfig(qengine)
+                    if qengine_is_onednn():
```

Comment:
can we add support for onednn in get_default_qconfig function?

Reply:
looks like it's support, wondering if we can just use it here

Reply:
Reduce-range is false by default for ONEDNN but here reduce-range needs to be true otherwise the test fails. We can update the comments below to make it clearer.

Reply:
I see, thanks